apps/studio/components/interfaces/Home/Connect/content/drizzle/env.tsx

```diff
@@ -0,0 +1,15 @@
+import SimpleCodeBlock from 'components/to-be-cleaned/SimpleCodeBlock'
+
+const ContentFile = () => {
+  return (
+    <div>
```

Comment:
the `<div>` seems redundant, could just wrap the container tag in wherever this content gets spat out. stop anyone screwing up later removing the `<div>` when updating these.